extensions/blocks/google-calendar/edit.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import {
+	Placeholder,
+	SandBox,
+	Button,
+	Notice,
+	ExternalLink,
+	PanelBody,
+} from '@wordpress/components';
+import { BlockIcon, InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { icon } from '.';
+import { isMobile } from '../../../_inc/client/lib/viewport';
```

Comment:
This seems like it should be exposed as a package.

Reply:
Yeah, pulling it like this may be a problem, as things may move around in the future. It also won't work on WordPress.com, since we don't have that folder there.

We have `@wordpress/viewport` as well.

Related discussion: p1581583668341200-slack-team-calypso

Reply:
Would be fairly straightforward to move Calypso's viewport to `packages` and publish to NPM if ya need: https://github.com/Automattic/wp-calypso/tree/a7dfb6f78b780cf71c01cd4e52dbc2239f961218/client/lib/viewport

I bet it's more similar (or same) to what we have in Jetpack.

Directly going for the core version is great of course.

Reply:
> Would be fairly straightforward to move Calypso's viewport to packages and publish to NPM

We're working on that right now in https://github.com/Automattic/wp-calypso/pull/39433

Reply:
replaced with @wordpress/viewport

Reply:
In the meantime, we now have `@automattic/viewport` just in case:
#13999